aws/sdk/examples/cloudformation/Cargo.toml

```diff
@@ -1,12 +1,17 @@
 [package]
-name = "cloudformation"
+name = "cloudformation-code-examples"
 version = "0.1.0"
-authors = ["Alistair McLean <mclean@amazon.com>"]
+authors = ["Alistair McLean <mclean@amazon.com>", "Doug Schwartz <dougsch@amazon.com>"]
 edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-env_logger = "0.8.2"
-cloudformation = {package = "aws-sdk-cloudformation", path = "../../build/aws-sdk/cloudformation"}
+cloudformation = { package = "aws-sdk-cloudformation", path = "../../build/aws-sdk/cloudformation" }
+aws-types = { path = "../../build/aws-sdk/aws-types" }
+
 tokio = { version = "1", features = ["full"] }
+
+env_logger = "0.8.2"
+structopt = { version = "0.3", default-features = false }
+tracing-subscriber = "0.2.18"
```

Comment:
linebreak?

Reply:
They don't hurt, but if it's non-idiomatic, I'll remove them.